### ISSUE TITLE
Implement `PATCH` for `/users/{payid}`

### ISSUE THREAD

**Issue by hbergren** [payid-org/payid#142]
Right now, it’s a huge hassle to update someone’s PayID. You have to `GET xpring.money:8081/hbergren$xpring.money`

Which gives you a payload like this:
```
{
  "payId": "hbergren$xpring.money",
  "addresses": [...],
}
```

And then you have to update the `payId` field in that response payload, and send it back to the server using:
`PUT xpring.money:8081/hbergren$xpring.money` (including the payload).

### Proposed Solution

What would be great is to have something like this:

```
PATCH xpring.money:8081/hbergren$xpring.money

{
  "payId": "hansbergren$xpring.money"
}
```

And have that just update the PayID of the user.

#### Requirements

- Updating the `payId` of a user should result in a `201 - Created` response
- Because the resource lives at a new place, we should include a `Location` header
- E2E tests for the `PATCH` method should be implemented
- We should require a request header `Content-Type: application/merge-patch+json`
- The Admin API server should set a `Accept-Patch: application/merge-patch+json` header.

**Comment by FlorentSG** [comment]
I am taking this one, I created the branch `fuzio-patch-user-payid-issue142` for it.

**Comment by hbergren** [comment]
Closed by #586
